patterns/ml-capacity-block/eks.tf

```diff
@@ -41,19 +41,6 @@ module "eks" {
   subnet_ids = module.vpc.private_subnets
 
   eks_managed_node_groups = {
-    # This node group is for core addons such as CoreDNS
-    default = {
-      instance_types = ["m5.large"]
-
-      min_size     = 1
-      max_size     = 2
-      desired_size = 2
-    }
-  }
-
-  # Note: ML capacity block reservations are only supported
-  # on self-managed node groups at this time
-  self_managed_node_groups = {
```

Comment:
Should we leave the self-managed example in there as well for those may need it?

Reply:
ya I guess it doesn't hurt to show both - let me update

Reply:
re-added in https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/1972/commits/193b135efb3fcc6cd98b75e921c74a5846499fb1

Reply:
:shipit: